Auth0/WebAuth.swift

```diff
@@ -243,3 +244,18 @@ public protocol WebAuth: Trackable, Loggable {
      */
     func clearSession(federated: Bool, callback: @escaping (Bool) -> Void)
 }
+
+public extension WebAuth {
+    
```

Comment:
The CI is not picking this up. However, on this line there is extra whitespace which should be removed.  I would suggest installing SwiftLint. SwiftLint is auto called in this Libraries build script and will pop up any warnings in the IDE. Thanks

Reply:
I have reverted the last commit and deleted the excess whitespace.